components/account/AccountHoverCard.vue

```diff
@@ -5,7 +5,7 @@ const { account } = defineProps<{
   account: mastodon.v1.Account
 }>()
 
-const relationship = useRelationship(account)
+const relationship = computed(() => account ? useRelationship(account).value : undefined)
```

Comment:
I don't think we should call `useRelationship` inside a `computed`. I think you should extract `useRelationship` to an outside `ref`. If you don't want that ref (that changes once `fetchRelationships()` finished), then you could abstract a helper around `requestedRelationships`

Reply:
done

Reply:
too many `useRelationship` usages to refactor it here: we should debonce the call